jenkins/jenkins_home/jobs/github-webhook/config.xml

```diff
@@ -59,7 +60,7 @@ For example by appending the expected parameters to a configurable URL, such as
         </org.jenkinsci.plugins.gwt.GenericVariable>
       </genericVariables>
       <regexpFilterText>$repoUrl $scmRef $scmRefForScheduledJobs</regexpFilterText>
-      <regexpFilterExpression>(refs/heads/master|refs/heads/main|refs/heads/develop|refs/heads/[0-9]+\.[0-9]+\.x)</regexpFilterExpression>
+      <regexpFilterExpression>(refs/heads/master|refs/heads/main|refs/heads/develop|refs/heads/[0-9]+\.[0-9]+\.x|main)</regexpFilterExpression>
```

Comment:
Looks like the scheduled job is not sending `refs/heads/main` but simply `main`, is that correct?

Reply:
Yes, that is correct.